packages/widgets/src/components/TradingInfoCard/TradingInfoCard.spec.tsx

```diff
@@ -0,0 +1,99 @@
+import userEvent from '@testing-library/user-event';
+import { screen, waitFor } from '../../../customTest';
+
+import { Icon } from '@ssa-ui-kit/core';
+import TradingInfoCard from './TradingInfoCard';
+
+describe('TradingInfoCard', () => {
+  it('Renders with value and title', () => {
+    render(<TradingInfoCard value={'400'} unit={'%'} title={'Title'} />);
+
+    screen.getByText('400');
+    screen.getByText('Title');
+  });
+
+  it('Renders with value, title, and unit', () => {
+    render(<TradingInfoCard value={'400'} unit={'%'} title={'Title'} />);
+
+    screen.getByText('400');
+    screen.getByText('%');
+    screen.getByText('Title');
+  });
+
+  it('Renders with value, title, unit, and icon', async () => {
+    render(
+      <TradingInfoCard
+        value={'400'}
+        unit={'%'}
+        title={'Title'}
+        icon={<Icon name="arrow-up" color="#2CA24D" size={16} />}
+      />,
+    );
+
+    screen.getByText('400');
+    screen.getByText('%');
+    screen.getByText('Title');
+
+    const icon = await screen.findByTitle(/Arrow Up/i);
+
+    expect(icon).toBeInTheDocument();
```

Comment:
This is redundant. `findByTitle` gets the promise rejected if the element isn't found.
https://testing-library.com/docs/queries/about/

Reply:
Done